docs/conf.py

```diff
@@ -225,7 +225,7 @@
 spelling_ignore_wiki_words = True
 spelling_show_suggestions = True
 spelling_word_list_filename = [
-    "spelling_wordlist.txt",
+    "../.config/dictionary.txt",
```

Comment:
Not sure if it's reasonable for the libencant dictionary to leak into some other tool.